newm-chain/docker-compose.yml

```diff
@@ -0,0 +1,63 @@
+version: "3.9"
+services:
+  newm-chain:
+    depends_on:
+      - postgres
+      - ogmios
+    image: newm-chain:latest
+    ports:
+      - "3737:3737"
+    environment:
+      - DATABASE_SERVER=postgres
+      - DATABASE_PORT=${DATABASE_PORT}
+      - DATABASE_NAME=${DATABASE_NAME}
+      - DATABASE_USERNAME=${DATABASE_USERNAME}
+      - DATABASE_PASSWORD=${DATABASE_PASSWORD}
+      - DATABASE_SALT=${DATABASE_SALT}
+      - DATABASE_SPENDING_PASSWORD=${DATABASE_SPENDING_PASSWORD}
+      - DATABASE_SHELLEY_GENESIS_HASH=${DATABASE_SHELLEY_GENESIS_HASH}
+      - OGMIOS_SERVER=ogmios
+      - OGMIOS_PORT=1337
+      - OGMIOS_SECURE=false
+      - OGMIOS_START_SLOT=${OGMIOS_START_SLOT}
+      - OGMIOS_START_HASH=${OGMIOS_START_HASH}
+      - JWT_SECRET=${JWT_SECRET}
+      - NEWM_CHAIN_SYNC_RAW_TXNS=${NEWM_CHAIN_SYNC_RAW_TXNS}
+      - NEWM_CHAIN_MONITOR_ADDRESSES=${NEWM_CHAIN_MONITOR_ADDRESSES}
+      - SENTRY_DSN=${SENTRY_DSN}
+      - SENTRY_ENVIRONMENT=${SENTRY_ENVIRONMENT}
+    #      - SSL_CERT_CHAIN_PATH=${SSL_CERT_CHAIN_PATH}
+    #      - SSL_PRIVATE_KEY_PATH=${SSL_PRIVATE_KEY_PATH}
+    restart: on-failure
+  postgres:
+    image: postgres:latest
+    restart: always
+    environment:
+      - POSTGRES_PASSWORD=password
+      - DATABASE_NAME=${DATABASE_NAME}
+      - DATABASE_USERNAME=${DATABASE_USERNAME}
+      - DATABASE_PASSWORD=${DATABASE_PASSWORD}
+    ports:
+      - "5432:5432"
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
+      - "./initdb.d/init-user-db.sh:/docker-entrypoint-initdb.d/init-user-db.sh"
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
+      interval: 1s
+
+  ogmios:
+    depends_on:
+      - postgres
+    image: cardanosolutions/cardano-node-ogmios:latest
```

Comment:
We probably don't want to use ogmios latest just yet. v6 is close to release, but I'm only about 30% done with making newm-chain compatible with it. v5.6.0 is the version we're compatible with currently.

Reply:
pinned to version `5.6.0_1.35.7`